web/wwwroot/js/site.js

```diff
@@ -25,14 +25,6 @@ function makeRequiredIfSelectedValueIsTarget(target, id, idToRequire) {
     }
 }
 
-function insertDefaultPostcodeIfNotSortByLocation(id) {

```

Comment:
was this meant to be removed?

Reply:
It didn't seem to be being used, so I removed it - but I've just check and it is being used (and I've also realised I slightly broke something else, so I'm going to fix that quickly!)